config/genericWords.json

```diff
@@ -3,7 +3,7 @@
     "^(barn|bazaa?r|bench|bou?tique|building|casa|church)$",
     "^(baseball|basketball|football|soccer|softball|tennis(halle)?)\\s?(field|court)?$",
     "^(club|green|out|ware)\\s?house$",
-    "^(driveway|el árbol|fountain|golf|government|graveyard)$",
+    "^(driveway|el árbol|fountain|generic|golf|government|graveyard)$",
```

Comment:
Will this be enough to exclude these "operators"?:
https://www.openstreetmap.org/node/1892277713
https://www.openstreetmap.org/node/878266316

Reply:
I was more unclear if that also covered the operator tag or just names.